packages/longhorn/longhorn-1.4/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,19 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,3 +1,16 @@
++annotations:
++  catalog.cattle.io/auto-install: longhorn-crd=match
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/display-name: Longhorn
++  catalog.cattle.io/kube-version: '>= 1.21.0-0'
```

Comment:
Ideally, this should be a closed ended range that excludes k8s 1.25 (`'>= 1.21.0-0 < 1.25.0-0'`). The reason being that regardless of longhorn supporting k8s 1.25, or longhorn having the PSPs disabled for 1.25, it will still be installable and have a different workflow than our other charts which may lead to confusion from an user standpoint. 

More info on how we are handling PSPs for 1.25: https://github.com/rancher/rancher/issues/40331

cc @prachidamle @MKlimuszka